helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -388,9 +385,6 @@ spec:
       - name: journallogpath
         hostPath:
           path: {{ .Values.fluentd.config.journalLogPath | quote }}
-      - name: secrets
-        secret:
-          secretName: {{ template "splunk-otel-collector.secret" . }}
```

Comment:
this is used for Otel collector as well for CA bundle. (https) line 361

Reply:
That one is different:`name: secret`